app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true
+  validates :content, presence: true
+  validates :user_id, presence: true
```

Comment:
Why do you validate the presence of the `user_id`?

Reply:
makes sense, I will omit that.